packages/animations/lib/src/open_container.dart

```diff
@@ -277,10 +268,11 @@ class _OpenContainerState<T> extends State<OpenContainer<T>> {
   Future<void> openContainer() async {
     final Color middleColor =
         widget.middleColor ?? Theme.of(context).canvasColor;
-    final T data = await Navigator.of(
+    final T? data = await Navigator.of(
       context,
       rootNavigator: widget.useRootNavigator,
-    ).push(_OpenContainerRoute<T>(
+    )!
+        .push(_OpenContainerRoute<T>(
```

Comment:
this formatting is very strange, is that what dartfmt produces?

Reply:
This is indeed produced by `flutter format .`